### ISSUE TITLE
Method for reset skintype to default.

### ISSUE THREAD

**Issue by milosic5** [HanSolo/Medusa#178]
Hi there,

if I set DASHBOARD SKIN for example and then change skin to PLAIN_AMP everything is ok,but when I change again to DASHBOARD it holds white background from previous skin.
This is just for example it is also problem with other skin parameters, not just background.

Is there any method for reset skin parameters to default or to clear parameters which previous skin was set?

Thank you in advance.

Sincerely
Milos

![dashboard_skin](https://user-images.githubusercontent.com/10159988/66122742-c4480500-e5e0-11e9-8d45-24648e7332c1.png)

![plain_amp](https://user-images.githubusercontent.com/10159988/66122740-c3af6e80-e5e0-11e9-96b1-d52fe95a852e.png)

![dashboard_skin_after_plain_amp_skin](https://user-images.githubusercontent.com/10159988/66122741-c4480500-e5e0-11e9-8399-6b3f85432df9.png)

**Comment by HanSolo** [comment]
Hmm...not really thought about changing skins at runtime but I already have a method to reset the default skin parameters. Will implement something to fix that issue, thx for the hint 👍🏻

**Comment by milosic5** [comment]
Ok, is that reset method (method name?) already in Medusa 11.2 or it would be committed?

**Comment by HanSolo** [comment]
Nope not yet, it will be in 11.3

**Comment by HanSolo** [comment]
The last commit contains a new public method on the Gauge class which is called reInit(). This method will reset the ui parameters. It will automatically be called when you switch the skin by calling setSkinType(). You could also call this method manually before you set another skin.

**Comment by milosic5** [comment]
On Maven repository is still 11.2 and I can't see changes in Gauge class.

**Comment by HanSolo** [comment]
yep...I did not create a new release yet, so the changes are only in the repo and not on maven central

**Comment by milosic5** [comment]
Ok, are you planned to increase version soon?

**Comment by HanSolo** [comment]
I usually wait before I create a new release to not create a release for every issue

**Comment by milosic5** [comment]
I understand, so anyway thank you for this commit and we will cooperate in the future for some improvement on your application.
